### PR TITLE
Upgrade steps for 0.15 -> 0.16

### PR DESCRIPTION
**Pull request by jnummelin**
Needs still some verification.

ping @kontena/core-platform please test these steps

### CODE REVIEW

File: docs/getting-started/installing/upgrading.md [hunk 1]
```diff
@@ -10,7 +10,8 @@ can be installed via rubygems.
 
 ## Versions
 
-* [0.14 to 0.15 (latest)](upgrading#upgrading-from-0-14-to-0-15)
+* [0.15 to 0.16 (latest)](upgrading#upgrading-from-0-15-to-0-16)
```
Comment (jakolehm) [inline docs/getting-started/installing/upgrading.md]: Actual title is wrong/missing so this anchor link does not work.

Comment (jnummelin) [inline docs/getting-started/installing/upgrading.md]: copy/paste issue, will fix

File: docs/getting-started/installing/upgrading.md [hunk 2]
```diff
@@ -10,14 +10,55 @@ can be installed via rubygems.
...
 * [0.8 to 0.9](upgrading#upgrading-from-0-8-to-9)
 
+### Upgrading from 0.15 to 0.16
+
+** Official Installation Method**
+
+- restart master:
+
+```
+$ sudo systemctl restart kontena-server-api
```
Comment (SpComb) [inline docs/getting-started/installing/upgrading.md]: mmm, the ubuntu_xenial packages use just `kontena-server`..

**Comment by jakolehm** [comment]
Works.

**Pull request merged**